### ISSUE TITLE
[Feature Request] Directional Terrain

### ISSUE THREAD

**Issue by HydrallUrakan** [ironmonk108/enhanced-terrain-layer#70]
On a few cases I've wanted difficult terrain to only apply in one direction - namely a literal uphill battle where it took double movement to go one direction, or a separate battle where the water had a current that made one direction even worse difficult terrain. Not entirely sure how best it could be implemented, though. There's a lot of angles at which one can enter a tile.

Perhaps slightly further than that, the ideal would be somehow setting things up so, like... This image would be plausible.
![image](https://user-images.githubusercontent.com/94702225/145282279-480d9e54-46e3-4714-80ca-cb3ff77a8514.png)

**Comment by KristjanLaane** [comment]
Yes, would "slope" difficult terrain would be great! Maybe could add some metadata as to the directions in which it applies.
